app/src/main/java/com/skgtecnologia/sisem/ui/SisemComposeApp.kt

```diff
@@ -34,7 +35,7 @@ fun ComposeApp() {
 
 private fun NavGraphBuilder.authGraph(modifier: Modifier) {
     navigation(
-        startDestination = AuthNavigationRoute.Login.route,
+        startDestination = AuthNavigationRoute.DeviceAuth.route,
```

Comment:
We can leave this one as is for now for testing purposes, but before merge we should revert